extra-modules/extra-mappings/src/main/kotlin/com/kotlindiscord/kord/extensions/modules/extra/mappings/stroage/MappingsConfig.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+@file:Suppress(
+    "UnderscoresInNumericLiterals",
+    "UndocumentedPublicClass",
+    "UndocumentedPublicFunction",
+    "UndocumentedPublicProperty",
+)
+
+package com.kotlindiscord.kord.extensions.modules.extra.mappings.stroage
```

Comment:
Did you mean to type `storage`? 🤔

Reply:
Wait ignore me I thought you were on about something else

Reply:
Fixed that